Core/Util/AudioFormatNEON.cpp

```diff
@@ -15,6 +15,8 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
+#ifdef __ARM_NEON__
```

Comment:
Yes and yes.
Also, __ARM_NEON__ is not always set (clang targeting aarch64 for Android), sometimes it's __ARM_NEON. Normalizing it to PPSSPP_ARCH(ARM_NEON) will help (as long as I fix the config file to use both).